tests/chez/chez027/StringParser.idr

```diff
@@ -38,13 +39,28 @@ maybeParser = do res <- optional (string "abc")
                  string "def"
                  pure $ isJust res
 
+-- test commaSep
+commaSepParser : ParseT IO (List Char)
+commaSepParser = do res <- commaSep alphaNum
```

Comment:
Can we just return `commaSep alphaNum` directly without the rewrapping here and below?

Reply:
Updated.